### ISSUE TITLE
Port for Evolution

### ISSUE THREAD

**Issue by danielquinn** [adam-p/markdown-here#310]
I'd like to use GNOME's Evolution mail client, but I find that I'm just too fond of the Markdown Here plugin in Thunderbird.  Would and Evolution port be all that difficult?

**Comment by sagikazarmark** [comment]
:+1: I would also be happy with that.

**Comment by TamasBarta** [comment]
This plugin is the only thing sticking me with Thunderbird. It's a pity that only Evolution has proper Exchange support.

**Comment by cheilman** [comment]
Agreed, I just switched to Evolution for mail because of some exchange issues, and I really miss being able to compose in Markdown.

**Comment by tflori** [comment]
I just found this thread on gnome: https://mail.gnome.org/archives/evolution-list/2011-July/msg00047.html

A markdown highlighting and conversion to a nicer html mail will make so much difference. It's faster to compose, easier to read in plain text, better formatted in html (no stupid differences between every paragraph). I'm really looking forward for such feature :+1:

**Comment by EmmanuelCharpentier** [comment]
I second this request.

Now, the real question is "How to implement this ? And who does it ?"

**Comment by marsh-circle** [comment]
Well -- looks like it was shut down by 'what on earth is MarkDown?', which took it off topic and seemed to get a straight 'No'.
It's a shame - I'm also held back by the lack of Markdown capabilities in Evolution. I've heard it's come a along way recently, but I've got used to it being easy to write emails without formatting issues.

**Comment by EmmanuelCharpentier** [comment]
The thread on the Gnome list dates back from 2011. I think it's safe to say that Markdown has gained a bit of recognition since.

The answers to "whhy not WYSIWIG ?" and "Why not HTML" are the same: "A loght markup language is bloody fucking **easier** and ***faster*** !".

Care for a second attempt ?

**Comment by marsh-circle** [comment]
@EmmanuelCharpentier I couldn't agree more - a WYSIWYG is a clumsy thing when compared to markup - hitting the marskown icon in thunderbird (courtesy of [Markdown Here](https://markdown-here.com/index.html) just makes it all really neat and tidy. I get comments from others in the office on email formatting. I haven't got time for that via WYSIWYG.

so what's the best plan to raise awareness...? would it be to raise a new issue, or is someine listening in here???

**Comment by TomSaw** [comment]
Many fast and easy to use markdown editors have popped up for Gnome Desktop and People like it.
It's time for Evolution to follow the herd!

**Comment by fugo** [comment]
with Evolution [3.44.1] now it's possible to use Markdown, Markdown as text or Markdown as HTML.
